settings.go

```diff
@@ -1,11 +1,75 @@
 package main
 
-import "os"
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
 
 var (
 	MOONSTREAM_ACCESS_TOKEN        = os.Getenv("MOONSTREAM_ACCESS_TOKEN")
 	MOONSTREAM_API_URL             = os.Getenv("MOONSTREAM_API_URL")
 	MOONSTREAM_API_TIMEOUT_SECONDS = os.Getenv("MOONSTREAM_API_TIMEOUT_SECONDS")
 
 	BUGOUT_ACCESS_TOKEN = os.Getenv("BUGOUT_ACCESS_TOKEN")
+
+	WAGGLE_CORS_ALLOWED_ORIGINS = os.Getenv("WAGGLE_CORS_ALLOWED_ORIGINS")
 )
+
+type ServerSignerConfig struct {
+	KeyfilePath         string `json:"keyfile_path"`
+	KeyfilePasswordPath string `json:"keyfile_password_path"`
+}
+
+// ReadConfig parses list of configuration file paths to list of Application Probes configs
+func ReadServerConfig(rawConfigPath string) (*[]ServerSignerConfig, error) {
+	var configs []ServerSignerConfig
+
+	configPath := strings.TrimSuffix(rawConfigPath, "/")
+	_, err := os.Stat(configPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, fmt.Errorf("file %s not found, err: %v", configPath, err)
+		}
+		return nil, fmt.Errorf("error due checking config path %s, err: %v", configPath, err)
+	}
+
+	rawBytes, err := os.ReadFile(configPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	configTemp := &[]ServerSignerConfig{}
```

Comment:
I prefer to use:

```go
configTemp := []ServerSignerConfig{}
unmarshalErr := json.Unmarshal(rawBytes, &configTemp)
```

This is the pattern we use everywhere else:

```
$ git grep Unmarshal
bugout.go:              parseErr := json.Unmarshal([]byte(result.Content), &reports[i])
cmd.go:                         parseErr := json.Unmarshal(batchRaw, &batch)
cmd.go:                                 rowParseErr := json.Unmarshal(jsonString, &batch[i])
cmd.go:                 parseErr := json.Unmarshal(messagesRaw, &messages)
moonstream.go:  unmarshalErr := json.Unmarshal(responseBody, &contracts)
moonstream.go:  unmarshalErr := json.Unmarshal(responseBody, &callRequests)
server.go:      err = json.Unmarshal(body, &req)
settings.go:    err = json.Unmarshal(rawBytes, configTemp)
```

Reply:
Two things:
1. Reference passed in `json.Unmarshal` call rather than declaring variable to be the reference.
2. Create a new error for unmarshalling to make the code more readable - `unmarshalErr` instead of `err`.

Reply:
I used this, because nodebalancer, monitoring, etc code written in this way. If it is important, I can follow as you wish.

Reply:
Also I am using `err` because:
```go
func abc(someInts []*int) (int, error) {
  var num *int
  for _, si := range someInts {
    // Here I forced to use `:=` but I want to be asure to use variable `num` scope from func `abc`
    num, numErr := someOperation(si)
     ...
  }
}
```

So I prefer this code:
```go
func abc(someInts []*int) (int, error) {
  var num *int
  var err error
  for _, si := range someInts {
    num, numErr = someOperation(si)
     ...
  }
}
```